src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -60,11 +59,12 @@ class DeployTask extends AbstractServerTask {
         })
     }
 
-    private Boolean container;
+    private Boolean container = null;
 
     @Option(option = 'container', description = 'Run the server in a Docker container instead of locally. The default value is false.')
     void setContainer(boolean container) {
         this.container = container;
+        super.@container = container;
```

Comment:
If you're setting `super.@container`, why does this file still need its own `private Boolean container` variable?

Reply:
Good question: because of DevTask. I'm keeping the implementation the same in DevTask and DeployTask.